packages/react-color-picker/tsconfig.build.json

```diff
@@ -1,4 +1,8 @@
 {
   "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "NodeNext",
+    "moduleResolution": "nodenext"
```

Comment:
This is a bug with the Rollup TS plugin I’m too lazy to fix right now—it doesn’t do a good job of tracing the `extends` to resolve the final TSConfig. This is redundant, but is easier than fixing the bug upstream.